docs/step-by-step/kubeapps-on-tkg/step-2.md

```diff
@@ -35,140 +31,152 @@ kubeops: # main backend APIs configuration
 
 assetsvc: # asset service configuration
 
-frontend: # nginx configuration (used to serve the dashboard and proxying the users' requests to the backend)
+frontend: # NGINX configuration (used to serve the dashboard and proxying user requests to the backend)
```

Comment:
We are not discussing these other parameters anywhere in this doc, should we remove them?

Reply:
Yes, I guess there are not useful for the main purpose of this guide and we already have them documented in the chart, so I'm ok removing them.